alica_tests/src/test/test_inherit_blackboard.cpp

```diff
@@ -0,0 +1,38 @@
+#include "alica_tests/TestWorldModel.h"
+#include "test_alica.h"
+#include <alica/test/Util.h>
+
+namespace alica
+{
+namespace
+{
+
+class TestInheritBlackboard : public AlicaTestFixture
+{
+protected:
+    const char* getRoleSetName() const override { return "Roleset"; }
+    const char* getMasterPlanName() const override { return "TestInheritBlackboardMaster"; }
+    bool stepEngine() const override { return false; }
+};
+
+TEST_F(TestInheritBlackboard, testInheritBlackboard)
+{
+    // Use inherited blackboards and check if keys are accessible
+    ae->start();
+    ae->getAlicaClock().sleep(alica::AlicaTime::milliseconds(100));
+    auto wm = dynamic_cast<alicaTests::TestWorldModel*>(ae->getWorldModel());
+    EXPECT_EQ(wm->passedParameters["masterKeyInBehavior"], 3);
+    EXPECT_EQ(wm->passedParameters["hasBehaviorKey"], 4);
```

Comment:
I don't get how this test is for inherited blackboards

Reply:
It checks if a key that is only available through inheritance is available.
The second check tests if a key defined in a child is not available in the parent when they are connected by inheritance. I have added comments in the code that explain that.

Reply:
I guess what I am confused about is the relationship between blackboards and the world model.  If this is a test for blackboards, why are we checking the world model?

Reply:
The blackboard itself is not accessible in the test code. The behaviors and plans read the blackboard and write into the world model. Then the test can check the world model. It's a pattern I found in several other tests and used again here.